script/plugin_webpack_directories.rb

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env ruby
+
+require 'bundler'
+require 'json'
+
+PLUGIN_NAME_REGEXP = /foreman*|katello*/
+
+# Only works for local dependencies
```

Comment:
Do you mean it only works for gems that have the source path set by `:path=>` in the Gemfile?